src/rdb.c

```diff
@@ -65,7 +65,7 @@ int rdbSaveType(rio *rdb, unsigned char type) {
     return rdbWriteRaw(rdb,&type,1);
 }
 
-/* Load a "type" in RDB format, that is a one byte unsigned integer.
+/* Load a "type" in RDB format, that is one byte unsigned integer.
```

Comment:
```suggestion
/* Load a "type" in RDB format, that is a one byte unsigned integer.
```
i think this one is wrong too